packages/carousel/src/Carousel.stories.js

```diff
@@ -81,7 +81,15 @@ function renderCards() {
   ))
 }
 
-const BasicTemplate = (args) => <Carousel {...args}>{renderCards()}</Carousel>
+const BasicTemplate = (args) => (
+  <Carousel
+    {...args}
+    onSlideClick={(props) => console.log('Slide click!', props)}
+    onSlideKeyDown={(props) => console.log('Keyboard click', props)}
```

Comment:
Mind turning these into actions for the stories? https://storybook.js.org/docs/react/essentials/actions#action-argtype-annotation

Reply:
Oops! Totally meant to do that, thanks for pointing it out!